raiden-ts/src/path/actions.ts

```diff
@@ -20,3 +21,13 @@ export const pathFindFailed = createStandardAction('pathFindFailed').map(
 export const pfsListUpdated = createStandardAction('pfsListUpdated')<{
   pfsList: readonly Address[];
 }>();
+
+export const udcBalanceFetch = createStandardAction('udcBalanceFetch')<{}>();
```

Comment:
These actions usually have a `meta` param to help identify the related request/response/error ones. Shouldn't you identify it somehow? Or else any following response/error could be associated to this request.

Reply:
I saw the meta, but in the end here you just fetch the balance of the `UserDeposit` contract for your address, so I couldn't think of any useful information that would go to meta.

Reply:
Hmm, if you're only fetching the balance and there's no useful meta, there's a big chance you don't need a set of actions for this, just calling the async method directly where it's needed may be good enough.

Reply:
Do you have an example in the code I could reference for this? I am trying to think how this would happen. The async method would not be an epic then right? it would be something different.

Reply:
Something like the `getTokenInfo` I would guess?

Reply:
Sure, look at this:
https://github.com/raiden-network/light-client/blob/27c60e432ac9ab978ebf3b3a75aa72917f0a35f9/raiden-ts/src/path/utils.ts#L80-L82
Notice this happens inside a normal function, but defines an observable, and therefore could as well be used inside an epic or mergeMap chain as a normal observable, which calls an async function and returns a result to be used on the next step of the chain.